docs/development/guide.md

```diff
@@ -0,0 +1,36 @@
+# Development Guide
+
+## Overview
+
+Read the document to find out how to install dependencies, launch the development environment, and run tests. Learn also what is the naming and architecture convention to follow in the project.
+
+> **NOTE:** This repository uses [Lerna](https://github.com/lerna/lerna) for managing local dependencies and better development experience.
+
+## Prerequisites
+
+Use the following tools to set up the project:
```

Comment:
The project is already set up and the only things needed to develop are node.js and npm, TS and React are dependencies, to be discussed